### PR TITLE
fix: `default-video-quality` patch crashing

### PR DESCRIPTION
**Pull request by Canny1913**
Requires:
- https://github.com/revanced/revanced-patches/pull/227

(osu i pressed "Reformat File" a thousand times)

### CODE REVIEW

File: app/src/main/java/app/revanced/integrations/patches/VideoQualityPatch.java [hunk 1]
```diff
@@ -127,6 +127,9 @@ public static void userChangedQuality(int selectedQuality) {
         userChangedQuality = true;
     }
 
+    public static void newVideoStarted(String useless) {
+        ReVancedUtils.setNewVideo(true);
```
Comment (oSumAtrIX) [inline app/src/main/java/app/revanced/integrations/patches/VideoQualityPatch.java:131]: Why is `ReVancedUtils` being used here? Shouldn't you set a field inside this class as you only need the video id here?

File: app/src/main/java/app/revanced/integrations/patches/VideoQualityPatch.java [hunk 2]
```diff
@@ -127,6 +127,9 @@ public static void userChangedQuality(int selectedQuality) {
         userChangedQuality = true;
     }
 
+    public static void newVideoStarted(String useless) {
```
Comment (oSumAtrIX) [inline app/src/main/java/app/revanced/integrations/patches/VideoQualityPatch.java:130]:
```suggestion
    public static void newVideoStarted(String videoId) {
```

File: app/src/main/java/app/revanced/integrations/patches/VideoQualityPatch.java (hunk 1 above)
Comment (Canny1913) [reply app/src/main/java/app/revanced/integrations/patches/VideoQualityPatch.java:131]: I only need to know if a new video is started or not. video id is useless in this patch.

Comment (oSumAtrIX) [reply app/src/main/java/app/revanced/integrations/patches/VideoQualityPatch.java:131]: Is there any use to call  `ReVancedUtils.setNewVideo(true);`, create a boolean field and work with that one.

Comment (Canny1913) [reply app/src/main/java/app/revanced/integrations/patches/VideoQualityPatch.java]: done

File: app/src/main/java/app/revanced/integrations/patches/VideoQualityPatch.java [hunk 3]
```diff
@@ -41,7 +42,7 @@ public static void changeDefaultQuality(int defaultQuality) {
     public static int setVideoQuality(Object[] qualities, int quality, Object qInterface, String qIndexMethod) {
         int preferredQuality;
         Field[] fields;
-        if (!ReVancedUtils.isNewVideoStarted() && !userChangedQuality || qInterface == null) {
+        if (!newVideo & !userChangedQuality || qInterface == null) {
```
Comment (oSumAtrIX) [inline app/src/main/java/app/revanced/integrations/patches/VideoQualityPatch.java:45]:
```suggestion
        if (!(newVideo || userChangedQuality) || qInterface == null) {
```

**Review by oSumAtrIX — CHANGES_REQUESTED** [review]

File: app/src/main/java/app/revanced/integrations/patches/VideoQualityPatch.java (hunk 3 above)
Comment (Canny1913) [reply app/src/main/java/app/revanced/integrations/patches/VideoQualityPatch.java:45]: wtf did i drink last night
tysm

**Comment by TheJeterLP** [comment]
please remove gradle.properties and changelog.md from this PR

**Comment by Canny1913** [comment]
semantic 💀

**Comment by bogadana** [comment]
just turn off actions or use a different branch

**Comment by TheJeterLP** [comment]
> semantic 💀

Disable actions on your PR

**Comment by Canny1913** [comment]
> please remove gradle.properties and changelog.md from this PR

reverted

File: app/src/main/java/app/revanced/integrations/videoplayer/VideoInformation.java [hunk 4]
```diff
@@ -1,6 +1,7 @@
 package app.revanced.integrations.videoplayer;
 
 import app.revanced.integrations.utils.LogHelper;
+import app.revanced.integrations.utils.ReVancedUtils;
```
Comment (oSumAtrIX) [inline app/src/main/java/app/revanced/integrations/videoplayer/VideoInformation.java:4]: Is this import still required?

```suggestion
```

Comment (Canny1913) [reply app/src/main/java/app/revanced/integrations/videoplayer/VideoInformation.java:4]: yes its used by sharedprefs

Comment (Canny1913) [reply app/src/main/java/app/revanced/integrations/videoplayer/VideoInformation.java:4]: no its not

**Pull request merged**